PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/functionapp/FunctionAppContextPublishProvider.kt

```diff
@@ -50,10 +50,10 @@ class FunctionAppContextPublishProvider : RiderContextPublishProvider {
         get() = RUN_CONFIG_NAME
 
     override fun getConfigurationForNode(project: Project,
-                                         projectModelNode: ProjectModelNode): Pair<RunConfiguration, ConfigurationFactory> {
+                                         entity: ProjectModelEntity): Pair<RunConfiguration, ConfigurationFactory> {
 
-        val projectData = RiderContextPublishProvider.getProjectDataRecursive(project, projectModelNode)
-                ?: error("Unexpected project node type. Cannot get project data for node ${projectModelNode.location}")
+        val projectData = RiderContextPublishProvider.getProjectDataRecursive(project, entity)
+                ?: error("Unexpected project node type. Cannot get project data for entity ${entity.url}")
```

Comment:
New entity has `getFileLocation()` method (or `getVirtualFileLocation()`). Does `url` give the same result? Can it be null for local files?

Reply:
Use url, it is OK. 
Have no idea about `getVirtualFileLocation` method although...